templates/contribute.html

```diff
@@ -0,0 +1,140 @@
+                        <p>Have you heard about an event happening in your city and you want to help? Contact the local <a href="http://djangogirls.org/events">organizing team</a> and ask if they need more coaches!</p>
+
+                        <p>You will be helping a group of 3 people to install what they need for the event and guide them through our <a href="http://tutorial.djangogirls.org/">tutorial</a>.</p>
+
+                        <p>Don't forget to check our <a href="http://coach.djangogirls.org/">Coaching Manual</a>!</p>
+                    </div>
+                </div>
+                <div class="col-md-6">
+                    <h3>Work on the website!</h3>
+                    <div class="cover">
+                        <a href="https://github.com/DjangoGirls/djangogirls">
+                            <img src="{% static 'img/global/contribute/debug.jpg' %}" alt="Computer screen with a console and a webpage." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>There is always something to do to make our website better. You can report (or fix!) bugs and typos, or work on new features.</p>
+
+                        <p>We are waiting for your <a href="https://github.com/DjangoGirls/djangogirls">pull requests!</a></p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="row clearfix">
+                <div class="col-md-6">
+                    <h3>Fix the tutorial!</h3>
+                    <div class="cover">
+                        <a href="https://github.com/DjangoGirls/tutorial">
+                            <img src="{% static 'img/global/contribute/tutoedit.jpg' %}" alt="A note book and a pen." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>Is something in the tutorial not clear, or is there a typo? You can improve the tutorial by making a pull request to our <a href="https://github.com/DjangoGirls/tutorial">Github repository</a>.</p>
+
+                        <p>You just finished our tutorial and want to help? Making pull requests to fix typo is a great way to use your new Git skills!</p>
+                    </div>
+                </div>
+                <div class="col-md-6">
+                    <h3>Translate the tutorial!</h3>
+                    <div class="cover">
+                        <a href="https://crowdin.com/project/django-girls-tutorial">
+                            <img src="{% static 'img/global/contribute/translate.jpg' %}" alt="A computer screen saying hello and golden balloons saying yay!." />
+                        </a>
+                    </div>
+                    <div class="info">
+                        <p>Our main tutorial is already translated in <a href="http://tutorial.djangogirls.org/">8 languages</a> but we always need help to maintain or start translating in a new language!</p>
```

Comment:
"Our tutorial has already been translated into 8 languages, but we always need helping maintaining those translations. Or you can help us start translating the tutorial into a new language!"